src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumOutputCollector.java

```diff
@@ -13,22 +16,22 @@
 
 import static org.openqa.selenium.support.locators.RelativeLocator.withTagName;
 
-public class SeleniumOutputCollector implements OutputCollector {
+public class FBASeleniumOutputCollector implements SeleniumOutputCollector {
     static final String OBJECTIVE_VALUE_LABEL_PATH = "//td[text()='Objective value']";
     static final String JOB_STATUS_BUTTON_PATH = "//button[@title='Job Status']";
     static final String OBJECTIVE_VALUE_VALUE_TAG_NAME = "td";
     static final String LOG_TEXT_CLASS_NAME = "kblog-text";
 
     private WebDriver webDriver;
 
-    public SeleniumOutputCollector(@Nonnull WebDriver webDriver) {
+    public FBASeleniumOutputCollector(@Nonnull WebDriver webDriver) {
         this.webDriver = webDriver;
     }
 
     @Nonnull
     @Override
-    public KBaseOutput collectOutput() {
-        return new KBaseOutput(getObjectiveValue(), getLogs());
+    public ApplicationOutput collectOutput(Job job) {
```

Comment:
Job is unused here for output. Will that change? Maybe some configurations on the job in the future specifying what the user would like to collect?

Reply:
That was my thinking, but we can remove it now and readd when we have a need later if we prefer. I'm assuming some collection will take longer than others (esp. if we download graphs), so if a particular batch is only interested in the performance we may be able to optimize with this.